app/controllers/api/v1/partner_requests_controller.rb

```diff
@@ -8,6 +8,10 @@ def create
     return head :forbidden unless api_key_valid?
 
     request = Request.new(request_params)
+    unless request&.partner&.approved?
+      request.status = "recertification_required"
+    end
+
     if request.save
       render json: request, status: :created
     else
```

Comment:
What happens if the `Partner` requires re-certification (setting the request status to `recertification_required`) -- will it still successfully save? Does the status get overridden?

Reply:
yeah... i had some things confused here. we've backed this change out in favor another mechanism to prevent partner requests if the partner is in the recertifaction status.